addons/l10n_it_edi/tests/test_edi_export.py

```diff
@@ -565,3 +565,34 @@ def test_export_invoice_with_two_downpayments(self):
         )
         invoice.action_post()
         self._assert_export_invoice(invoice, 'test_export_invoice_with_two_downpayments.xml')
+
+    def test_send_prezzo_unitario_converted_to_company_currency(self):
+        """
+        Test that the prezzo unitario is converted to the company currency when the invoice is in a foreign currency
+        """
+        usd = self.env['res.currency'].search([('name', '=', 'USD')])
+        self.env['res.currency.rate'].create({
+            'name': '2025-01-01',
+            'rate': 2,
+            'currency_id': usd.id,
+            'company_id': self.company.id,
+        })
+
+        invoice = self.env['account.move'].with_company(self.company).create({
+            'move_type': 'out_invoice',
+            'invoice_date': '2022-03-24',
+            'invoice_date_due': '2022-03-24',
+            'partner_id': self.italian_partner_a.id,
+            'partner_bank_id': self.test_bank.id,
+            'currency_id': usd.id,
+            'invoice_line_ids': [
+                Command.create({
+                    'name': 'standard_line',
+                    'price_unit': 100,
+                    'quantity': 1,
+                    'tax_ids': [Command.set(self.default_tax.ids)],
+                }),
+            ],
+        })
+        invoice.action_post()
```

Comment:
Wasn't it possible to use the `init_invoice` helper? https://github.com/odoo/odoo/blob/2ffbfd34a704ba2faf28cdb8113168e2da7b37e3/addons/account/tests/common.py#L430